libs/drape_frontend/scenario_manager.cpp

```diff
@@ -92,7 +92,7 @@ void ScenarioManager::ThreadRoutine()
       CenterViewportAction * centerViewportAction = static_cast<CenterViewportAction *>(action.get());
       m_frontendRenderer->AddUserEvent(make_unique_dp<SetCenterEvent>(
           centerViewportAction->GetCenter(), centerViewportAction->GetZoomLevel(), true /* isAnim */,
-          false /* trackVisibleViewport */, nullptr /* parallelAnimCreator */));
+          false /* ignoreVisibleViewport */, nullptr /* parallelAnimCreator */));
```

Comment:
Well, I'm ok with the change to keep ```ignoreVisibleViewport == false``` (as a default behaviour) here and above.
But need to test these use-cases.